src/modules/top-level.nix

```diff
@@ -69,10 +69,33 @@ in
     stdenv = lib.mkOption {
       type = types.package;
       description = "The stdenv to use for the developer environment.";
-      default = pkgs.stdenv;
+      default =
+        if pkgs.stdenv.isDarwin
+        then
+          pkgs.stdenv.override
```

Comment:
This override should probably be applied to `config.stdenv` since changing stdenv will invalidate this override

Reply:
Yeah, that's probably better.